gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/BloodPressureValidatorUtil.java

```diff
@@ -94,4 +96,10 @@ public static boolean containsValidBloodPressureTriple(RCMRMT030101UK04CompoundS
 
         return false;
     }
+
+    public static boolean isBloodPressureWithBatteryAndBloodPressureTriple(
+        RCMRMT030101UK04CompoundStatement compoundStatement) {
+        return BATTERY_VALUE.equals(compoundStatement.getClassCode().get(0))
+            && containsValidBloodPressureTriple(compoundStatement);
```

Comment:
I wonder if `containsValidBloodPressureTriple` can be made private now? Looks like this is the public method now.

Reply:
unfortunately it is still used in multiple places, one for a future refactor I believe